### PR TITLE
Integrate erp

### PR DESCRIPTION
**Pull request by lebaz20**
Fixes comicrelief/technology-planning#36

### CODE REVIEW

File: config/webpack.config.dev.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ module.exports = {
       inject: true,
       template: paths.appHtml,
       cspPlugin: { //adds CSP Meta tag plugin
-        enabled: true,
+        enabled: false,
```
Comment (lebaz20) [inline config/webpack.config.dev.js:174]: local testing didn't work with CSP enabled, as it rejects all requests to non-production domains

File: src/pages/GiftAid/utils/Utils.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ export const getFormValues = (validation, urlId = null, update = false) => {
     }
     // set Giftaid choice for update form
     if (key === 'giftAidClaimChoice') {
-      fieldValues.confirm = value; // reassign to confirm field
+      fieldValues.confirm = value * 1; // reassign to confirm field
```
Comment (lebaz20) [inline src/pages/GiftAid/utils/Utils.js:77]: sometimes it's int, sometimes string, I wanted to make sure it has one type (int now)

Comment (AndyEPhipps) [reply src/pages/GiftAid/utils/Utils.js:77]: Out of interest, why `* 1` rather than using `parseInt`?

**Review by AndyEPhipps — APPROVED** [review]

**Comment by lebaz20** [comment]
*1 is shorter to write, but I guess parseInt is easier to read, so will use it instead.

**Review by AndyEPhipps — APPROVED** [review]

**Pull request merged**